packages/sdk/src/index.ts

```diff
@@ -6,4 +6,7 @@ export { FailedHomeError } from './error';
 
 export { NomadContext } from './NomadContext';
 
-export * from './messages/types';
```

Comment:
Were the types being used by other packages that use the sdk?

Reply:
Addressed in the new commit